airflow/executors/base_executor.py

```diff
@@ -147,7 +147,6 @@ def heartbeat(self):
         self.sync()
 
     def change_state(self, key, state):
-        self.log.debug("Changing state: %s", key)
```

Comment:
Unrelated change here (I usually try to avoid these.)

Reply:
👍